src/components/o-modal/_preview.html

```diff
@@ -0,0 +1,76 @@
+<!--Please create here a HTML example by using just default HTML tags-->
+
+<button onclick="openModal()">Open modal</button>
+
+<axa-modal open>
```

Comment:
i would default wise not have it open as it will overlap over all other components: http://localhost:3000/organisms.html